app/views/state_file/questions/shared/_md_retirement_income_deductions_review_header.erb

```diff
@@ -0,0 +1,39 @@
+<% if current_intake.eligible_1099rs.length.positive? %>
+  <section id="retirement-income-source" class="white-group">
+    <div class="spacing-below-5">
+      <h2 class="text--body text--bold spacing-below-15"><%=t(".retirement_income_deductions") %></h2>
+
+      <% current_intake.eligible_1099rs.each_with_index do |state_file1099_r, index| %>
```

Comment:
note that we're changing to `eligible_1099rs` instead of `state_file1099_rs` here -- problem navigating back to a specific 1099R unless we have the same indices or a given 1099r!

Reply:
in fact, we _do_ have a problem. I used frank rollover & tried to navigate to the 2nd retirement deduction shown -- it takes me to oops page (on demo; fixed on this PR)